### PR TITLE
Add support for Carthage (iOS)

### PR DESCRIPTION
**Pull request by AndrewSB**

### CODE REVIEW

**Comment by blender** [comment]
Awesome! :+1:

**Comment by blender** [comment]
I have added a OSX target and made a pull request on @AndrewSB 's fork

**Comment by AndrewSB** [comment]
@blender your changes look good, merged :)

**Comment by AndrewSB** [comment]
hey @syedhali!
love the work you've done, any chance you've had a chance to look at these changes yet?

Thanks,
Andrew

File: EZAudio.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				GCC_C_LANGUAGE_STANDARD = gnu99;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```
Comment (syedhali) [inline EZAudio.xcodeproj/project.pbxproj]: Just one tweak, could you set the iOS deployment target to 6.0? Is there anything that would require the minimum to be 8.0?

**Comment by syedhali** [comment]
Looks good! Just have one comment above

File: EZAudio.xcodeproj/project.pbxproj (hunk 1 above)
Comment (blender) [inline EZAudio.xcodeproj/project.pbxproj]: Frameworks are only supported from iOS 8 on.

Comment (syedhali) [inline EZAudio.xcodeproj/project.pbxproj]: Ah that's right!

**Comment by blender** [comment]
:tada: :tada: :tada: :tada: :tada:

**Comment by syedhali** [comment]
Thank you @AndrewSB @blender!

**Comment by AndrewSB** [comment]
😀😀😀

**Pull request merged**